src/nanodbc.cpp

```diff
@@ -849,17 +849,15 @@ class connection::connection_impl
                 enable_async(event_handle);
         #endif
 
-        NANODBC_SQLCHAR dsn[1024];
-        SQLSMALLINT dsn_size = 0;
         NANODBC_CALL_RC(
             NANODBC_FUNC(SQLDriverConnect)
             , rc
             , conn_
             , 0
             , (NANODBC_SQLCHAR*)connection_string.c_str(), SQL_NTS
-            , dsn
-            , sizeof(dsn) / sizeof(NANODBC_SQLCHAR)
-            , &dsn_size
+            , NULL
```

Comment:
> NULL should read nullptr

There's quite a few instances of NULL instead of nullptr across the codebase (probably most of them from me, sorry about that). Do you mind if I leave this as-is for now and correct them all in a separate PR?